ffi/diplomat/c/include/ICU4XUnicodeScriptMapPropertyResult.h

```diff
@@ -0,0 +1,24 @@
+#ifndef ICU4XUnicodeScriptMapPropertyResult_H
```

Comment:
Is there a reason to have this in a separate file from `ICU4XUnicodeScriptMapProperty.h`, or is it just an implementation detail of Diplomat?

Reply:
Implementation detail of Diplomat.